CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to Undate - A fuzzy date Python library
+
+
+Hey there!
+
+If you found your way here that probably means you are curious about how to contribute to this project. This is great! We are always looking for new contributors. If you can't find the information you are looking for in this document or anywhere else in the repo, please consider [opening a ticket](https://github.com/dh-tech/hackathon-2022/issues) so we know there is something we need to address.
+
+## Project Setup
+Instructions on how to set up the project locally and how to run the tests can be found [in the Readme file](https://github.com/dh-tech/hackathon-2022/blob/main/README.md).
```

Comment:
Is it possible to make this a relative link so it won't break when we rename the repo?

Reply:
Relative links only seem to work for other files in the repo, not for issues, pull requests etc 😕